core/src/main/java/com/scalar/db/transaction/consensuscommit/CoordinatorGroupCommitter.java

```diff
@@ -7,7 +7,7 @@
 import java.util.concurrent.ThreadLocalRandom;
 
 public class CoordinatorGroupCommitter
-    extends GroupCommitter<String, String, String, String, Snapshot> {
+    extends GroupCommitter<String, String, String, String, String, Snapshot> {
```

Comment:
From here there will be many changes, but most of them are just adding an additional generic type (`EMIT_KEY` -> `EMIT_PARENT_ID` and `EMIT_FULL_ID`)